.php_cs

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+$finder = PhpCsFixer\Finder::create()
+    ->in('src')
+    ->notPath('Kernel.php');
```

Comment:
Why are you ignoring Kernel? It's just another class, should be properly formatted.

Reply:
ok why not !!!

Reply:
resolve with https://github.com/DarkmiraTour/community-event-manager/pull/91/commits/8b81715f79d91e957a72feeaba10b6ffe15283bd